packages/@react-spectrum/progress/docs/ProgressBar.mdx

```diff
@@ -0,0 +1,121 @@
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/progress';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {ProgressBar} from '@react-spectrum/progress';
+```
+
+# ProgressBar
+
+<p>{docs.exports.ProgressBar.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ProgressBar']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/bar-loader/'}
+  ]} />
+
+## Example
+
+```tsx example
+<ProgressBar label="Loading…" value={50} />
+```
+
+## Value
+
+By default Progress Bars are controlled with the `value` prop representing current percentage (as an integer).
+In this case, the minimum and maximum values default to 0 and 100, respectively.
+
+```tsx example
+<ProgressBar label="Loading…" value={25} />
+```
+
+Alternatively, a different scale can be used by setting the `minValue` and `maxValue`.
+
+```tsx example
+<ProgressBar label="Loading…" minValue={50} maxValue={150} value={100} />
+```
+
+Values are formatted by default as a percentage, but can modified via `formatOptions` for different formats.
+The `formatOptions` are displayed via the [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat) applied for the current locale.
+
+```tsx example
+<ProgressBar label="Loading…" formatOptions={{style: 'currency', currency: 'JPY'}} value={60} />
+```
+
+### Accessibility
+
+Progress Bars are made accessible as per the ["progressbar" role in WAI-ARIA 1.1](https://www.w3.org/TR/wai-aria-1.1/#progressbar).
```

Comment:
Maybe not needed? We haven't linked to the spec in other docs pages... Definitely will from @react-aria docs.